### ISSUE TITLE
[enterprise-4.9] Issue in file monitoring/configuring-the-monitoring-stack.adoc

### ISSUE THREAD

**Issue by briantward** [openshift/openshift-docs#43227]
### Which section(s) is the issue in?

https://docs.openshift.com/container-platform/4.9/monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack_configuring-the-monitoring-stack

### What needs fixing?

yaml for the prometheus volume needs to be corrected. Incorrect spacing starting at "spec"...

```
data:
  config.yaml: |
    prometheusK8s: 
      volumeClaimTemplate:
       spec:
         storageClassName: fast
         volumeMode: Filesystem
         resources:
           requests:
             storage: 40Gi
```

**Comment by cbremble** [comment]
@bburt-rh would you mind reviewing this one and if applicable, add the monitoring-issue label? Thank you!

**Comment by cbremble** [comment]
Added to Jira: https://issues.redhat.com/browse/RHDEVDOCS-4348 for finalizing the PR

**Comment by bburt-rh** [comment]
This is now fixed and merged into the 4.10+ docs. The changes should be live on the site within a few hours.
